docs/paas+/dev/onboarding-to-paas+.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Onboarding to PaaS+
+description: Get your project ready and start upgrading with PaaS+
+template: concept-topic-template
+---
+
+Onboarding to PaaS+ happens in two steps:
+
+1. Getting your project ready and compliant with a set of pre-requirements to ensure you fully benefit from all the features.
+2. Getting started with Spryker CI.
+
+
+## 1. Get your project ready for PaaS+  
+
+Before you can start managing the upgrades of your project with PaaS+, fulfill the following prerequisites.
+
+### 1.1. Update each module to the latest version
+
+Spryker upgrader provides automatic minor and patch updates for the current version of each module. Since we release most of the updates for the latest versions of modules, if a module is not of the latest major version, it will not receive most of the updates.
+
+### 1.2. Implement Spryker SDK in development
+
+Using [Spryker SDK](/docs/sdk/dev/spryker-sdk.html) for development helps you achieve compliance with Spryker architecture and customization best practices.
+
+### 1.3 Make your code compliant with Spryker Quality Gate
+
+Spryker Quality Gate contains code checkers that ensure code compliance with Srpyker development and customization guidelines. When an application is compliant with the guidelines, it can take minor and patch updates without breaking anything, even if it’s highly customized.
+
+For instructions on making your code compliant with the guidelines, see [Keeping a project upgradable](/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html).
+
+### 1.4. Implement E2E testing into your project's development workflow
+
+Upgrades are provided as PRs that are automatically created in a project’s repository. To make sure all business functionality works as expected, you will need to review and test the PRs before merging.
+
+### 1.5. Migrate the codebase to GitHub or GitLab
+
+At the moment, upgrades are provided only for repositories on GitHub or GitLab. If you want to use a different git provider, reach out to your CSM, so we can add it to our product roadmap.
+
+### 1.6. Optional: Implement headless design
+
+Currently, the quality gate does not evaluate frontend customizations. You can either move to headless or apply frontend upgrades manually.
+
+### 1.7 Migrate to Spryker Cloud Commerce OS
```

Comment:
I put it here because of "it sounds like a lot of work" if put as the first requirement. On-premise customers might think "I'm not qualified for it, so no reason to read further. But your call anyway.

Reply:
Ok -it makes sense. Let's leave it.